federation/pkg/federation-controller/util/federated_informer.go

```diff
@@ -96,6 +96,13 @@ type FederatedInformer interface {
 	Stop()
 }
 
+// FederatedInformer with extra method for setting fake clients.
```

Comment:
If this is test only, we should have it in test-only dir, so that we are not linking it to production code.